src/lib/Room/Room.vue

```diff
@@ -466,6 +469,12 @@ export default {
       this.scrollMessagesCount = 0
       this.resetMessageSelection()
 
+      if (!isNaN(parseInt(this.roomId)) && !this.overlayInitialized) {
+        const uploadOverlay = new FileUploaderOverlay()
+        uploadOverlay.initEventListeners()
```

Comment:
```suggestion
        uploadOverlay.init()
```

Geralmente a inicialização de um componente/classe ou o que for exige vários passos. Por acaso aqui só temos os event listeners, mas poderia ter mais (talvez até no futuro).

Por isso, usar algo mais genérico como `init()` já deixa isso pronto. Agora esse método só vai ter a chamada para o `initEventListeners()`, mas já fica pronto para crescimento alterações.

Reply:
Ajustado